pkg/asset/installconfig/azure/client.go

```diff
@@ -152,3 +157,32 @@ func (c *Client) getProvidersClient(ctx context.Context) (azres.ProvidersClient,
 	client.Authorizer = c.ssn.Authorizer
 	return client, nil
 }
+
+// GetDiskSkus checks if the diskType exists for the given region.
+func (c *Client) GetDiskSkus(ctx context.Context, region string) ([]azsku.ResourceSku, error) {
+	client := azsku.NewResourceSkusClient(c.ssn.Credentials.SubscriptionID)
+	client.Authorizer = c.ssn.Authorizer
+	ctx, cancel := context.WithTimeout(ctx, 30*time.Second)
+	defer cancel()
+
+	var regionPages []azsku.ResourceSku
+
+	for skuPage, err := client.List(ctx); skuPage.NotDone(); err = skuPage.NextWithContext(ctx) {
+		if err != nil {
+			return nil, errors.New(fmt.Sprintf("Error fetching pages from Azure : %s", err.Error()))
```

Comment:
https://godoc.org/github.com/pkg/errors#Errorf

don't use uppercase to start an error, it doesn't look good when wrapped.

Reply:
also this could easily just be `errors.Wrap(err, "fetching resource SKUs")`